src/cmd/services/m3aggregator/main/main.go

```diff
@@ -93,6 +94,11 @@ func main() {
 	httpAddr := cfg.HTTP.ListenAddress
 	httpServerOpts := cfg.HTTP.NewServerOptions()
 
+	// Get the m3msg server options.
+	m3msgServerScope := scope.SubScope("m3msg-server").Tagged(map[string]string{"server": "m3msg"})
```

Comment:
hm probably want to call this passthrough-server or some such?

also don't think you need the tags in addition to the subscope